napari/layers/_tests/_utils.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+
+def compare_dicts(dict1, dict2):
```

Comment:
I do not like that this function is not covered. On the other hand, I'm not sure if it is a proper reason to add tests for these function.